### PR TITLE
Issue #GEN-112: Changed urls for activity

### PR DESCRIPTION
**Pull request by tuj**

### CODE REVIEW

File: web/modules/custom/itk_activity/itk_activity.routing.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+  path: '/aktivitet/{nid}/signup'
   defaults:
     _controller: '\Drupal\itk_activity\Controller\ActivityController::signup'
     _title: 'Sign up'
   requirements:
     _permission: 'access itk_activity user signup'
 
 itk_activity.external_signup_form:
-  path: '/activity/{nid}/external_signup'
+  path: '/aktivitet/{nid}/external_signup'
```
Comment (magnify) [inline web/modules/custom/itk_activity/itk_activity.routing.yml]: "external_signup" is a system URL? Is it visible to the user?

**Review by magnify — CHANGES_REQUESTED** [review]
Look at comment about "external_signup".

File: web/modules/custom/itk_activity/itk_activity.routing.yml (hunk 1 above)
Comment (tuj) [reply web/modules/custom/itk_activity/itk_activity.routing.yml]: It is visible to the user.

changed to    /tilmeld   og   /tilmeld_ekstern

File: web/modules/custom/itk_activity/itk_activity.routing.yml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+  path: '/aktivitet/{nid}/tilmeld'
   defaults:
     _controller: '\Drupal\itk_activity\Controller\ActivityController::signup'
     _title: 'Sign up'
   requirements:
     _permission: 'access itk_activity user signup'
 
 itk_activity.external_signup_form:
-  path: '/activity/{nid}/external_signup'
+  path: '/aktivitet/{nid}/tilmeld_ekstern'
```
Comment (magnify) [inline web/modules/custom/itk_activity/itk_activity.routing.yml]: Use dash (-) instead of underscore (_)

**Review by magnify — CHANGES_REQUESTED** [review]

File: web/modules/custom/itk_activity/itk_activity.routing.yml (hunk 2 above)
Comment (tuj) [reply web/modules/custom/itk_activity/itk_activity.routing.yml]: Done

**Review by magnify — APPROVED** [review]

**Pull request merged**